src/app.jsx

```diff
@@ -428,7 +369,7 @@
 
     const onDoubleClickNavigate = useCallback((file) => {
         const newPath = [...path, file.name].join("/");
-        if (file.type === "directory" || file.to === "directory") {
+        if (file.type === "dir" || file.to === "dir") {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-210-20240129-100325-cbe55367-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)